extensions/sql-database-projects/resources/templates/newTsqlExternalStreamingJobTemplate.sql

```diff
@@ -5,6 +5,5 @@
 
 EXEC sys.sp_create_streaming_job @NAME = '@@OBJECT_NAME@@', @STATEMENT = 'INSERT INTO SqlOutputStream SELECT
```

Comment:
two quick things,
1. please check that after replacng the @@vars with values the project builds/deploys fine
2.  when using references like SqlOutputStream  - check once how View template refers to Table - and lets be consistent on that pattern...

Reply:
One more nit - the file names in the snapshot above are the name that you entered or that we get by default? - if default - we should have them consistent too. Lets have suffix 1 on all or none

Reply:
1. Yup, the template builds, validates, and deploys without any changes from the user.

2. Sorry, what pattern of how we do references?

3. Agreed, and fixed (see updated screenshot)